net.newpipe.NewPipe.metainfo.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>net.newpipe.NewPipe</id>
+  <name>NewPipe</name>
+  <summary>Free and lightweight YouTube frontend for Android</summary>
+  <project_license>GPL-3.0-or-later</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
missing developer https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#developer-name

Reply:
added